### PR TITLE
Add go-consistent linter.

### PR DESCRIPTION
**Pull request by AlekSi**
That's an experiment. Let's use `make check-style` during development in separate branches to get some experience and then decide if we want it on Travis CI and in other repos.

### CODE REVIEW

File: commands/base.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ var customLabelRE = regexp.MustCompile(`^([a-zA-Z_][a-zA-Z0-9_]*)=([^='", ]+)$`)
 // E.g. the value of [[--custom-labels='region=us-east1, mylabel=mylab-22']] will be received by this function
 // as [[region=us-east1, mylabel=mylab-22]].
 func ParseCustomLabels(labels string) (map[string]string, error) {
-	result := make(map[string]string)
+	result := map[string]string{}
```
Comment (BupycHuk) [inline commands/base.go:111]: why did you change it? go suggests to use make to init maps. https://blog.golang.org/go-maps-in-action

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: commands/base.go (hunk 1 above)
Comment (AlekSi) [reply commands/base.go:111]: Well, this post explains how maps work, but not suggest any particular style. For example:
```
The same syntax may be used to initialize an empty map,
which is functionally identical to using the make function:

m = map[string]int{}
```

go-consistent works by examining all packages and suggesting variants that are more often used in them: https://github.com/quasilyte/go-consistent#overview `pmm-admin` just happened to use map[K]V{} syntax more often.

In fact, personally, I prefer `make(map[K]V)` syntax. So I started to change pmm-admin to use that style for it to be more frequent and consistent. Then I encountered this: https://github.com/percona/pmm-admin/blob/08412685ea76d93336093fb53b8dbe62899d4316/commands/base_test.go#L34-L40 `make` on line 39 would be really strange. So I decided against it and converted all code to the more common style.

And again, that's an experiment to get some first-hand experience. We may as well make it completely optional in the future.

**Review by BupycHuk — APPROVED** [review]
Let's try

**Comment by AlekSi** [comment]
Let's keep `map[K]V{}` in mind. I'm still not buying it.
